src/main/java/org/apache/datasketches/kll/KllHelper.java

```diff
@@ -65,24 +65,24 @@ static int computeTotalCapacity(final int k, final int m, final int numLevels) {
   }
 
   /**
-   * Returns the capacity of a specific level.
+   * Returns the item capacity of a specific level.
    * @param k the accuracy parameter of the sketch. Maximum is 2^29.
    * @param numLevels the number of current levels in the sketch. Maximum is 61.
    * @param height the zero-based index of a level with respect to the smallest level.
    * This varies from 0 to 60.
-   * @param minWidth the minimum level width. Default is 8.
+   * @param m the minimum level width. Default is 8.
    * @return the capacity of a specific level
    */
-  static int levelCapacity(final int k, final int numLevels, final int height, final int minWidth) {
+  static int levelCapacity(final int k, final int numLevels, final int height, final int m) {
     assert (k <= (1 << 29));
     assert (numLevels >= 1) && (numLevels <= 61);
     assert (height >= 0) && (height < numLevels);
     final int depth = numLevels - height - 1;
-    return (int) Math.max(minWidth, intCapAux(k, depth));
+    return (int) Math.max(m, intCapAux(k, depth));
```

Comment:
Is there a reason that you renamed `minWidth` to m, which conveys less information than the former version?

Reply:
The variable "m" is used everywhere else in the sketch and is defined multiples places as well.  I see no need to have another variable name that means the same thing, which adds confusion.  (How is minWidth different from m?) The answer is that it is not!